phpBB/styles/prosilver/theme/forms.css

```diff
@@ -210,19 +204,20 @@ fieldset.display-options a {
 
 /* Display actions for ucp and mcp pages */
 fieldset.display-actions {
-	line-height: 2em;
+	line-height: 22px;
 	text-align: right;
 	white-space: nowrap;
-	padding-right: 1em;
+	padding-right: 11px;
 }
 
 fieldset.display-actions label {
 	white-space: nowrap;
 	padding-right: 2px;
 }
 
+/* Not used anywhere */
```

Comment:
Might be left over from previously when we had a different type of dropdown for the sort options.